products/kubernetes.md

```diff
@@ -23,6 +23,13 @@ auto:
 
 # Support and EOL dates can be found on https://kubernetes.io/releases/patch-releases/#detailed-release-history-for-active-branches
 releases:
+  - releaseCycle: "1.34"
+    releaseDate: 2025-08-27
+    eoas: ""
+    eol: ""
```

Comment:
wasn't able to determine the eol dates of 1.34 reading the page linked above (I've created https://github.com/kubernetes/website/issues/52085 for tracking the same and can do a follow-up when it's updated)